index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ernest Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+    <!--Head Section-->
+    <header>
+        <div class="logo">
+            <h1 id="logo">My Logo</h1>
+        </div>
+        <div class="menu">
+            <a href="#">
+                <img src="./images/Icon-Menu.png" alt="Menu" class="menu-icon"/>
+            </a>
+        </div>
+    </header>
+    <!--Main Section-->
+    <main>
+        <h1>I'm Ernest Glad to see you!</h1>
+        <p class="main-paragraph">I am a Web developer! I can help you build your website. Kindly go through my work samples! If you are interested and you have any website to develop, do contact me. I would appreciate recommendation as well.</p>
+        <p class="connect">LET'S CONNECT</p>
+    </main>
+    <!--Footer Section-->
+    <footer>
+        <nav>
+            <ul>
+                <li>
+                    <a href="#" class="social twiter">
+                        <img src="./images/twiter.png" alt="Twitter"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social linkendIn">
+                        <img src="./images/Linkedinicon.png" alt="LinkendIn"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social twiter">
+                        <img src="./images/mail.png" alt="Email"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social github">
+                        <img src="./images/Vector.png" alt="GitHub"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social peace">
+                        <img src="./images/fingers.png" alt="Peace"/>
+                    </a>
+                </li>
+            </ul>
+        </nav>
```

Comment:
- [ ] Great job with the `ul` but I don't recommend wrapping it in a `nav` tag because the nav is used for the links that helps navigate through the app